aldryn_faq/south_migrations/0024_create_default_faq_config.py

```diff
@@ -0,0 +1,186 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+
+from django.conf import settings
+from django.db import models, connection, transaction
+
+
+def create_placeholders(app_config, orm):
+    """
+    Creates placeholder instances for each Placeholder field on provided
+    app_config.
+    """
+    from cms.models import Placeholder
+
+    for field in app_config._meta.fields:
+        if field.__class__ != models.fields.related.ForeignKey:
+            # skip not FK fields
+            continue
+
+        if not (field.rel.to == Placeholder or
+                field.rel.to == orm['cms.Placeholder']):
+            # skip other fields.
+            continue
+
+        placeholder_name = field.name
```

Comment:
I disagree:

    ipdb> field.slotname
    u'faq_content'
    ipdb> field.name
    u'placeholder_faq_content'
    ipdb> field.slotname in dir(app_config)
    False
    ipdb> field.name in dir(app_config)
    True

and we are checking if ``app_config.placeholder_faq_content`` (FK) exists by checking the ``_id`` property (``app_config.placeholder_faq_content_id``) in this example case.